### DISCUSSION TITLE
Is there a way to make it ignore the state of a binary sensor unless the state is X for X minutes?

### DISCUSSION THREAD

**Discussion by angrycatmeowmeow** [kalkih/mini-graph-card#1104] [Q&A] [answered]
Perhaps my use case will better explain what I'm trying to do. I have a vibe sensor on my dryer I use in automations to tell me when it's done. Is there any way to make it only graph an "on" point if the sensor has seen vibration for greater than 5 minutes? Every time we bump the dryer getting dog snacks it registers as "on", and it's just ugly on my dashboard. I could probably get around this with helpers and automations, but it would be cleaner if I could just do it in the card.

![image](https://github.com/kalkih/mini-graph-card/assets/67968528/6821f9a3-d66c-4660-98c4-fdc2e5a95530)

**Comment by ildar170975** [comment, marked as answer]
> an "on" point if the sensor has seen vibration for greater than 5 minutes? 

Consider using a template sensor for this, frontend cards are not supposed to do this.
